changes/ee/feat-13861.en.md

```diff
@@ -0,0 +1,14 @@
+A new configuration item `rule_engine.db_actions_undefined_vars_as_null` has been added to the rule engine to ensure that when performing rule databases actions, undefined variables are treated as NULL.
+
+The following actions and SQL functions are affected by this configuration item:
+
+Actions:
+- MySQL
+- PostgreSQL
```

Comment:
Nit. It is actually unclear to me how it affects Postrgres connector, does not look like there would be any difference.

Reply:
fixed